webpack/assets/javascripts/foreman_navigation.test.js

```diff
@@ -1,97 +0,0 @@
-import $ from 'jquery';
-import { init } from './foreman_navigation';
```

Comment:
why deleting the entire file? should we not have tests for the functions in foreman_navigation?

Reply:
those were test for the `tfm.nav.*` funcs that no longer exists (`init() activate() markActive()`)

I consulted @sharvit about this file and we agreed that we shouldn't test `store.dispatch` because those actions are being tested in their own test file.